src/pocketmine/event/entity/EntityDamageEvent.php

```diff
@@ -159,6 +159,7 @@ public function __construct(Entity $entity, $cause, $damage){
 							}
 						}
 					}
+					$this->useArmors();
```

Comment:
This is one reason I tell people not to use the web editor. That method does not exist in EntityDamageEvent nor any of its parents.